examples/tti/tti_example.py

```diff
@@ -23,10 +23,11 @@ def setup(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
 
     model = IGrid(origin, spacing,
                   true_vp,
-                  .4*np.ones(dimensions),
-                  -.1*np.ones(dimensions),
-                  -np.pi/7*np.ones(dimensions),
-                  np.pi/5*np.ones(dimensions))
+                  rho=None,
```

Comment:
No, it's in my acoustic, but for tti, there is no point adding more paramterers untill we get the "simple" case `rho=1` running in a decent time. It will be as astable a s the current one just a bit more costly computationally.